src/main/java/sirius/db/mixing/Property.java

```diff
@@ -463,7 +463,8 @@ public Object getValue(Object entity) {
      * @return the extracted and translated value to include in messages shown to the user
      */
     public String getValueForUserMessage(Object entity) {
-        return NLS.toUserString(getValue(entity));
+        final String userString = NLS.toUserString(getValue(entity));
+        return Strings.isEmpty(userString) ? NLS.get("Property.emptyValue") : userString;
```

Comment:
I'd move this check to BaseEntity.java:146 (and BaseEntity.java:169) (maybe via a helper method) - as otherwise, each subclass of Property has to take care of this check manually...

Reply:
I agree -> changed